app/assets/javascripts/discourse/routes/unknown.js.es6

```diff
@@ -1,8 +1,27 @@
 import DiscourseRoute from "discourse/routes/discourse";
+import RSVP from "rsvp";
 import { ajax } from "discourse/lib/ajax";
 
 export default DiscourseRoute.extend({
-  model() {
-    return ajax("/404-body", { dataType: "html" });
+  model(params) {
```

Comment:
very nitpick but given you use params only for path, you could write:

```
model({ path } = params) {

and then later do:  `data : { path }`

Reply:
eslint complained when I tried to do this.. how are you supposed to spell the parameter?

Reply:
hmmm weird thought this should work, nevermind. Maybe I will try when pushed.